docs/versions/unversioned/sdk/background-fetch.md

```diff
@@ -17,13 +16,18 @@ Returns a promise resolving to one of these values:
 -   `BackgroundFetch.Status.Denied` - The user explicitly disabled background behavior for this app or for the whole system.
 -   `BackgroundFetch.Status.Available` - Background updates are available for the app.
 
-### `BackgroundFetch.registerTaskAsync(taskName)`
+### `BackgroundFetch.registerTaskAsync(taskName, options)`
 
-Registers background fetch task with given name. Registered tasks are saved in persistent storage and restored once the app is initialized. Remember to also call [BackgroundFetch.setMinimumIntervalAsync](#backgroundfetchsetminimumintervalasyncminimuminterval) as fetch operations will not be occurring with its default value.
+Registers background fetch task with given name. Registered tasks are saved in persistent storage and restored once the app is initialized.
 
 #### Arguments
 
--   **taskName (_string_)** -- Name of the task to register. The task needs to be defined first - see [TaskManager.defineTask](../task-manager/#taskmanagerdefinetasktaskname-task) for more details.
+-   **taskName (_string_)** -- Name of the task to register. The task needs to be defined first - see [TaskManager.defineTask](../task-manager#taskmanagerdefinetasktaskname-task) for more details.
+-   **options (_object_)** -- An object of options:
+    -   **minimumInterval (_number_)** -- Inexact interval in seconds between subsequent repeats of the background fetch alarm. The final interval may differ from the specified one to minimize wakeups and battery usage.
+    On Android it defaults to **15 minutes**. On iOS it calls [BackgroundFetch.setMinimumIntervalAsync](#backgroundfetchsetminimumintervalasyncminimuminterval) behind the scenes and the default value is the smallest fetch interval supported by the system (**10-15 minutes**).
+    -   **stopOnTerminate (_boolean_)** -- Whether to stop receiving background fetch events after user terminates the app. Defaults to `true`. (**Android only**)
+    -   **startOnBoot (_boolean_)** -- Whether to restart background fetch events when the device has finished booting. Defaults to `false`. (**Android only**)
```

Comment:
Is there a way to add a bit more information about the `stopOnTerminate`?
What does exactly means "after user terminates the app"?

Reply:
background app: switch to another
terminate app: close it